README.md

```diff
@@ -86,6 +94,43 @@ import pybamm_cookie
 
 pybamm_cookie.__version__
 ```
+### Project Layout
+```bash
+.
+├── .github
+│   └── workflows
+├── docs
+│   └── _static
+├── src
+│   ├── pybamm_example_project
+│   ├── models
+│   │   └── input
+│   │  
+│   └─ parameters
+│       └── input
+│  
+├── tests
+│    ├── generated_project_tests
+│    │  
+│    └── user_tests
+├─ noxfile.py
+└─ pyproject.toml
```

Comment:
It would be good to enumerate other folders as well (`workflows/`, `input/`, ..., or just add `└── ...`)
```suggestion
├── .github
│   └── workflows
├── docs
│   └── _static
├── src
│   ├── pybamm_example_project
│   ├── models
│   │   └── input
│   └─ parameters
│       └── input
├── tests
│    ├── generated_project_tests
│    └── user_tests
├─ noxfile.py
└─ pyproject.toml
```